privatelink/azure/terraform/cluster.tf

```
@@ -0,0 +1,22 @@
+resource "confluent_kafka_cluster" "dedicated" {
+  display_name = "${var.cluster_name}"
+  availability = "MULTI_ZONE"
+  cloud        = "AZURE"
+  region       = "${var.region}"
+  dedicated {
+    cku = "${var.cluster_cku}"
+  }
+
+  environment {
+    id = data.confluent_environment.env.id
+  }
+  
+  network {
+    id = confluent_network.azure-private-link.id
```

Private link itself also has id. We want to be clear that the id here is the azure private link network id. 
```suggestion
    id = confluent_network.azure-private-link-network.id
```